pb_plugins/dcsdkgen/type_info.py

```diff
@@ -0,0 +1,50 @@
+import json
+from os import environ
+
+
+class TypeInfo:
+    _default_types = {
+        1: "double",
+        2: "float",
+        4: "uint64_t",
+        5: "int",
+        8: "bool",
+        9: "std::string",
+        13: "uint32_t"
```

Comment:
I wonder where we need `uint32_t`.

Reply:
Not sure. I just translated the types I knew (i.e. the ones we use), but it would make sense here to have all the proto types (it's a plugin for the proto idl after all).

Reply:
I would like to constrain the types that we need a bit rather than having all of them.

Reply:
Well we don't "have them". The plugin knows how to translate a `uint32_t`, that's all. What we do with that is defined by our proto files. In the same way, we don't fork protobuf to not "have `uint32_t` there. We just don't use it in our proto files, that's it.